### DISCUSSION TITLE
Resets on Sonoff Basic R2

### DISCUSSION THREAD

**Discussion by TxBillbr** [arendst/Tasmota#12167] [Q&A] [answered]
Suggestions for how to troubleshoot. Latest firmware on Sonoff Wifi Smart Switch (Board labeled Sonoff RF R2 POWER V1.3) on first startup is in reset/boot loop. Firmware loaded with Tazmotizer 1.2 Blue LED flashes at 1s interval 50% duty cycle. Tried minimal, but could not get a successful flash.

--- From the console ---
 ets Jan  8 2013,rst cause:1, boot mode:(3,4)

load 0x4010f000, len 3584, room 16
tail 0
chksum 0xb0
csum 0xb0
v3969889e
~ld

00:00:00.001 HDW: ESP8285
00:00:00.050 CFG: Loaded from flash at FB, Count 9
00:00:00.055 QPC: Count 1
00:00:00.062 Project tasmota Tasmota Version 9.4.0(tasmota)-2_7_4_9(2021-04-23T10:07:22)
00:00:00.214 WIF: WifiManager active for 3 minutes
00:00:00.722 HTP: Web server active on tasmota_FAF7E9-6121 with IP address 192.168.4.1
00:00:06.937 QPC: Reset

**Comment by sfromis** [comment, marked as answer]
Absolutely never flash -minimal as first binary unless you want a bricked device. It is 100% normal that Tasmota devices will start in a 3-minute reset/boot loop till you connect to the AP and configure. Or you could configure basic details from Tasmotizer or serial console.

**Comment by TxBillbr** [comment]
Thanks for the prompt information. I had not seen that with my previous 
Sonoff R2 POWs. I've managed to get a couple of the new ones working since 
the post.
